server/server.js

```diff
@@ -73,11 +74,47 @@ module.exports = function(port, googleAuthoriser) {
         res.json(getTweets());
     });
 
+    app.get("/api/test/hashtag", function(req, res) {
+        var result = [];
+        var query = {
+            q: hashtags[1],
+            count: 2
+        };
+        client.get("search/tweets", query, function(error, tweets, response) {
+            if (tweets) {
+                tweets.statuses.forEach(function(tweet) {
+                    result.push(tweet);
+                });
+                res.json(result);
+            } else {
+                console.log(error);
+            }
+        });
+    });
+
+    function getTweetsWithHashtag() {
+        hashtags.forEach(function (hashtag) {
```

Comment:
This will make a twitter API request for each hashtag - can you make one request querying for any of the hashtags? I think the suggested limit is ten in one query